codecov.yml

```diff
@@ -1,6 +1,25 @@
 codecov:
   require_ci_to_pass: false
 
+ignore:
```

Comment:
it's sad that there is no option to use `.gitignore` to ignore files out of the box

Reply:
yes :'(